Documentation/operations/upgrade.rst

```diff
@@ -322,6 +322,21 @@ Annotations:
   ``reserved:remote-node``. The change might impact those who have a network policy
   allowing access to the service from inside the cluster, and the policy was used
   to allow access from outside.
+* The default metrics exporter ports has changed to reserved ports in the `Prometheus project <https://github.com/prometheus/prometheus/wiki/Default-port-allocations>`_.
+
+  +-----------------------+-----------------------+-------------------------+
+  | Exporter              | Default port          | Default port            |
+  |                       | Cilium <= v1.11       | Cilium => v1.12         |
```

Comment:
```suggestion
  |                       | Cilium <= v1.11       | Cilium >= v1.12         |
```

Thanks for the table by the way, nice way to visualise the changes quickly.

Reply:
Looks like this change got missed in the latest update?

Reply:
:thinking:  Humm - I will have an other look at why